src/OrchardCore.Modules/OrchardCore.Tenants/Controllers/AdminController.cs

```diff
@@ -167,11 +175,22 @@ public async Task<IActionResult> Index(TenantIndexOptions options, PagerParamete
             };
 
             // We populate the SelectLists
+            model.Options.TenantsCategories = allSettings
+                .GroupBy(t => t["Category"])
+                .Where(t => !String.IsNullOrEmpty(t.Key))
+                .Select(t => new SelectListItem(t.Key, t.Key))
```

Comment:
```
model.Options.TenantsCategories = allSettings
    .GroupBy(t => t["Category"])
    .Where(t => !String.IsNullOrEmpty(t.Key))
    .Select(t => new SelectListItem(t.Key, t.Key, String.Equals(model.Category, t.Key, StringComparison.OrdinalIgnoreCase)))
    .ToList();
```

where model.Category = Selected Category, and you set whether the `SelectListItem` is selected or not

Reply:
Go it, you want to set th select property, I will update the PR